src/common/containers/build-details.js

```diff
@@ -72,9 +71,12 @@ class BuildDetails extends Component {
         <BetaNotification />
         <Breadcrumbs>
           { user &&
-            <Link to={`/user/${user.login}`}>My repos</Link>
+            <BreadcrumbsLink to={`/user/${user.login}`}>My repos</BreadcrumbsLink>
           }
-          <Link to={`/user/${repository.fullName}`}>{repository.fullName}</Link>
+          {' ' /* vanilla assumes space between breadcrumb elements */}
```

Comment:
Styles for Vanilla breadcrumbs pattern assume there will be a space between breadcrumb items, but JSX doesn't render space between elements, so we need to manually add that.

But was reported to Vanilla about it: https://github.com/vanilla-framework/vanilla-framework/issues/1146